tensorflow/core/kernels/cwise_ops_test.cc

```diff
@@ -25,9 +25,9 @@ namespace tensorflow {
 
 // Creates a Graph which applies a unary "func" on a 3D float tensor
 // of "num" elements.
-static Graph* Unary(const string& func, int num) {
+static Graph* Unary(const string& func, int num, DataType dtype) {
   Graph* g = new Graph(OpRegistry::Global());
-  Tensor data(DT_FLOAT, TensorShape({64, 64, num / (64 * 64)}));
+  Tensor data(dtype, TensorShape({64, 64, num / (64 * 64)}));
   CHECK_GT(data.NumElements(), 0);
   data.flat<float>().setRandom();
```

Comment:
Oops I am sorry. I can send a patch in  a few hours.

> On 27.07.2016, at 01:51, yifeif <notifications@github.com> wrote:
> 
> In tensorflow/core/kernels/cwise_ops_test.cc:
> 
> >    Graph* g = new Graph(OpRegistry::Global());
> > -  Tensor data(DT_FLOAT, TensorShape({64, 64, num / (64 * 64)}));
> > +  Tensor data(dtype, TensorShape({64, 64, num / (64 * 64)}));
> >    CHECK_GT(data.NumElements(), 0);
> >    data.flat<float>().setRandom();
> Can we have a patch PR to fix this?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>